package.json

```diff
@@ -25,11 +25,14 @@
     "prepare": "husky install"
   },
   "devDependencies": {
-    "@babel/preset-env": "^7.19.4",
+    "@babel/core": "^7.23.5",
+    "@babel/preset-env": "^7.23.5",
     "@babel/preset-typescript": "^7.18.6",
     "@changesets/cli": "^2.26.2",
+    "@types/babel__core": "^7",
     "@types/jest": "^29.5.4",
     "@types/lodash": "^4.14.191",
+    "@types/node": "^20.10.3",
```

Comment:
This @types/node dependency was necessary but missing, causing yarn install error.